### ISSUE TITLE
Add NVDA Remote to the core of NVDA

### ISSUE THREAD

**Issue by slannon97** [nvaccess/nvda#9263]
### Describe the solution you'd like
NVDA Remote is extremely useful. It should be integrated into the core version of NVDA so that everyone has access to it right out of the box after installing the screen reader. It would make it much easier for folks to use because it would be built into the screen reader and ready for immediate use instead of requiring the somewhat advanced knowledge of installing it from a website.

**Comment by DrSooom** [comment]
For security reasons I wouldn't prefer this. A NVDA add-on can be even faster updated instead of NVDA itself. And a quick reaction on new (security related) bugs is highly necessary for this type of add-ons.

**Comment by Andre9642** [comment]
Duplicate of #4390
Agree with @DrSooom

**Comment by Brian1Gaff** [comment]
I do however feel that adjustments to this add on considering it was crowd 
funded and therefore may be subject to different licensing has been an 
issue. It will I think need substantial changes very soon anyway.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by Adriani90** [comment]
I am voting for implementing this as an optional feature in NVDA's core. This is because people in professional environments really need this feature and we need sustainable support and maintaining. We will see if the developer of the remote addon will react accordingly when NVDA will change to Python 3. but hopefully we have this implemented until then. As of now, I am closing this as duplicate.
